.github/workflows/tests.yml

```diff
@@ -37,11 +37,11 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.6', '3.7']
+        python-version: ['3.7', '3.8', '3.9']
```

Comment:
If there isn't CI running on 3.6 I'm reluctant to say setupmeta is compatible wiuth it.  Am I reading this right?

Reply:
Tests pass with 3.6, but there is some unrelated failure on py3.6 venv creation currently on GH actions. setupmeta's is still py3.6 compatible, but it is getting harder to exercise py3.6 indeed (including locally, where you can't use `tox-uv` for exameple, must use tox v3)

Reply:
Bumped `python_requires` to 3.7+. You're right, no point in trying to keep 3.6 around any longer.